profiles.md

```diff
@@ -0,0 +1,235 @@
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }],
+    "dc.description.provenance": [{
+        "value": "Made available in DSpace on 2020-07-20T15:31:11Z (GMT). No. of bitstreams: 0",
+        "language": "en",
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }],
+    "dc.identifier.uri": [{
+        "value": "http://localhost:4000/handle/123456789/511",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }],
+    "dspace.entity.type": [{
+        "value": "Person",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }]
+  },
+  "inArchive": true,
+  "discoverable": true,
+  "withdrawn": false,
+  "lastModified": "2020-07-20T16:02:19.608+0000",
+  "type": "item"
+}
+```
+It would respond with:
+* 200 OK - Returning the item if there's a match
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions to view the item
+* 404 Not found - if the item doesn't exist
+
+### EPerson
+**GET /api/eperson/profiles/<:eperson-uuid>/eperson**
+
+Returns the eperson owning the profile. The JSON response document is as follow
+
+```json
+{
+  "id": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "uuid": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "name": "msmith@samltest.id",
+  "handle": null,
+  "metadata": {
+    "eperson.firstname": [{
+        "value": "Mortimer",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }],
+    "eperson.lastname": [{
+        "value": "Smith",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }]
+  },
+  "netid": "msmith@samltest.id",
+  "lastActive": "2019-10-25T08:05:46.530+0000",
+  "canLogIn": true,
+  "email": "msmith@samltest.id",
+  "requireCertificate": false,
+  "selfRegistered": false,
+  "type": "eperson"
+}
+```
+
+It would respond with:
+* 200 OK - Returning the eperson if there's a match
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions to view the eperson
+* 404 Not found - if the eperson doesn't exist
+
+## Creating a new Profile
+**POST /api/eperson/profiles?eperson=<:eperson-uuid>**
+
+Create a new profile for the specified person or for the current user. This action can only be done by the EPerson themselves or an Administrator.
+
+Parameters:
+* eperson-uuid (optional): the uuid of eperson for which the profile is to be created; if not specified, the eperson is the authenticated one
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions
+* 409 Conflict - if for the specified eperson a profile already exists
```

Comment:
As I noted in my review on the REST implementation PR, I'm starting to think this should also be a 422.  This seems similar to how you cannot map an Item to the same Collection twice, and if you do, that also returns a 422.